onnxscript/ir/_convenience.py

```diff
@@ -0,0 +1,93 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+"""Convenience methods for constructing and manipulating the IR.
+
+This is an internal only module. We should choose to expose some of the methods
+after they are proven to be useful.
+"""
+
+from __future__ import annotations
+
+from typing import Any, Mapping, Sequence
+
+from onnxscript.ir import _core, _protocols
+
+
+def convert_attributes(attrs: Mapping[str, Any]) -> list[_core.Attr]:
```

Comment:
Is this the same as [the one in _tape.py](https://github.com/microsoft/onnxscript/blob/5a0cd8ef59179e8e53221c7a53fbf437c42c942a/onnxscript/rewriter/_tape.py#L15) ? We can presumably use this there?

Reply:
Sure! I will create a follow up